src/App.test.jsx

```diff
@@ -0,0 +1,9 @@
+import { test } from 'vitest'
+import { render, screen } from '@testing-library/react'
+import { App } from './App'
+
+test('renders title on navbar', () => {
```

Comment:
Mettons en place une bonne strucuture : `describe -> test` ou `describe -> it`

la différence entre les deux est sur le wording : `it should do` / `test if condition`